Cargo.toml

```diff
@@ -42,30 +42,29 @@ thiserror = "1.0.30"
 tide = { version = "0.16.0", optional = true }
 tokio-crate = { package = "tokio", version = "1.17.0", features = ["rt"], optional = true }
 tracing = "0.1.32"
-trust-dns-resolver = "0.20"
+trust-dns-resolver = "0.21.2"
 void = "1.0.2"
 
 [dependencies.libp2p]
-version = "0.43.0"
+version = "0.48.0"
 default-features = false
 features = [
```

Comment:
You might want to enable "rsa" feature since that was moved into its own feature (https://github.com/libp2p/rust-libp2p/pull/2860) otherwise you might run into errors if you use libp2p bootstrap or dialing any rsa keys (https://github.com/libp2p/rust-libp2p/issues/2971). If its not needed, could also move it into a optional feature too.

Reply:
Thanks for catching this! An optional feature looks like the idiomatic way to go.